src/OpenTelemetry.PersistentStorage.FileSystem/DirectorySizeTracker.cs

```diff
@@ -40,6 +40,16 @@ public DirectorySizeTracker(long maxSizeInBytes, string path)
 
     public void FileRemoved(long fileSizeInBytes) => Interlocked.Add(ref this.directoryCurrentSizeInBytes, fileSizeInBytes * -1);
 
+    /// <summary>
+    /// Checks if the space is available for new blob.
+    /// </summary>
+    /// <remarks>
+    /// This method is not thread safe and may give false positives/negatives.
+    /// This is acceptable as the file write can be retried if needed.
```

Comment:
Could you specifically mention why is this acceptable for false positive vs false negative?

Reply:
done